test/test_helper.rb

```diff
@@ -1,5 +1,10 @@
 # frozen_string_literal: true
 
```

Comment:
Let's add a comment explaining why we do it in 3.1? Also should we change this to be a strict equal?

Reply:
Will do!

I initially tried doing a strict equal but the RUBY_VERSION is in the format 3.1.x - I'll change the check to `RUBY_VERSION.start_with?("3.1")` because I agree that this should be more strict.